WebRoot/js/zimbraMail/prefs/view/ZmFilterRulesView.js

```diff
@@ -79,20 +76,17 @@ function() {
 											 context:ZmId.VIEW_FILTER_RULES});
 		this._toolbar.replaceElement(toolbarEl);
 		this._tabGroup.addMember(this._toolbar);
+		//add `ZAltButton` style to 'Create Filter' toolbar button
+		var addFilterBtn = this._toolbar.getButton(ZmOperation.ADD_FILTER_RULE);
+		addFilterBtn && addFilterBtn.delClassName('ZToolbarButton','ZButton ZAltButton'); //to look-it like ZButton
```

Comment:
can we do this when creating a button, that way we don't have to remove existing class and add new